src/main/java/com/develhope/spring/vehicle/controller/VehicleController.java

```diff
@@ -0,0 +1,25 @@
+package com.develhope.spring.vehicle.controller;
+
+
+import com.develhope.spring.vehicle.service.VehicleService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/vehicle")
+public class VehicleController {
+
+    @Autowired
+    private VehicleService vehicleService;
+
+
+
+
+    @DeleteMapping("/{id}")

```

Comment:
Tutto il controller di conseguenza cambia in funzione di ciò che è stato scritto nel service